xml/System.Net.Http/HttpRequestOptions.xml

```diff
@@ -134,7 +134,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Removes all items from the <see cref="T:System.Collections.Generic.ICollection`1" />.</summary>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
        <remarks>
          <format type="text/markdown"><![CDATA[

## Remarks

This member is an explicit interface member implementation. It can be used only when the <xref:System.Net.Http.HttpRequestOptions> instance is cast to an <xref:System.Collections.Generic.ICollection`1> interface.

          ]]></format>
        </remarks>
```